cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocProvisioner.java

```diff
@@ -143,7 +144,15 @@ public Cluster createCluster(ProvisionerContext context) throws Exception {
                  numWarnings, numWarnings > 1 ? "s" : "",
                  String.join("\n", createOperationMeta.getWarningsList()));
       }
+      DataprocUtils.emitMetric(context, conf.getRegion(), null,
+                               // Constants.Metrics.Provisioner.CREATE_CLUSTER_COUNT =
```

Comment:
No need to have these comment. If you want to define these as constants, define a `DataprocConstants` class in the dataproc module.

Reply:
removed comments.